product-mini/platforms/nuttx/wamr.mk

```diff
@@ -62,6 +66,35 @@ else ifeq (${WAMR_BUILD_TARGET}, XTENSA)
   CFLAGS += -DBUILD_TARGET_XTENSA
   INVOKE_NATIVE := invokeNative_xtensa.s
   AOT_RELOC := aot_reloc_xtensa.c
+else ifeq (${WAMR_BUILD_TARGET}, RISCV64)
+
+ifeq (${CONFIG_ARCH_FPU},y)
+  CFLAGS += -DBUILD_TARGET_RISCV64_LP64F
+  INVOKE_NATIVE += invokeNative_riscv64_lp64d.s
+else ifeq (${CONFIG_ARCH_DPFPU}, y)
+  CFLAGS += -DBUILD_TARGET_RISCV64_LP64D
+  INVOKE_NATIVE += invokeNative_riscv64_lp64d.s
+else
+  CFLAGS += -DBUILD_TARGET_RISCV64_LP64
+  INVOKE_NATIVE += invokeNative_riscv64_lp64.s
+endif
+
+  AOT_RELOC :=
+
+else ifeq (${WAMR_BUILD_TARGET}, RISCV32)
+
+ifeq (${CONFIG_ARCH_FPU}, y)
+  CFLAGS += -DBUILD_TARGET_RISCV64_ILP32F
```

Comment:
Same as above, no asm src file is added here. Do you want to implement the related codes to support LP64F and ILP32F?

Reply:
Yes, and AOT support should also be considered

Reply:
Great! Do you need any help from us?